src/UI/Components/ServiceInstanceForm/Components/SelectFormInput.tsx

```diff
@@ -28,15 +32,50 @@ export const SelectFormInput: React.FC<Props> = ({
   handleInputChange,
   ...props
 }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
   const selectOptions = Object.keys(options).sort();
   const formattedOptions = selectOptions.map((option) => {
     return {
-      value: option,
+      value: options[option],
       children: option,
-      isSelected: selectOptions.length === 1 || option === attributeValue,
+      isSelected:
+        selectOptions.length === 1 || options[option] === attributeValue,
     };
   });
 
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (
```

Comment:
component and its internal functions could get docstrings to align with the new habits

Reply:
added